sass/themes/cr/2019/components/membership-signup/js/membership-signup.js

```diff
@@ -1,5 +1,11 @@
 (function($) {
   $(document).ready(function() {
+
+    // remove donate button if memebership row is visible
+    if($('.paragraph--membership-signup').is(':visible')) {
+      $('.donate').hide();
```

Comment:
I wonder if this `.donate` could be a more specific selector, just in case..

Reply:
yes will change it... also moved the JS to CR instead of inside the component...

Reply:
👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍